test/expected.txt

```diff
@@ -112,6 +129,14 @@
 || 
 || 3 persons listed!
 || ===================================================
+|| Enter command: || [Command entered:  sort]
```

Comment:
Like what @niloc94 has already mentioned, the test inputs are already sorted and as such would not be very useful.